test-suite/Math/NumberTheory/TestUtils.hs

```diff
@@ -153,6 +153,18 @@ testIntegralPropertyNoLarge name f = testGroup name
   , QC.testProperty "quickcheck Natural" (f :: wrapper Natural -> bool)
   ]
 
+-- Tests for 'Int', 'Word' are omitted because 'inverseSigmaK/inverseJordan'
+-- tests would quickly oveflow in these types.
```

Comment:
Since this helper is specific to `InverseTests`, let's move it there. Also its name is a bit confusing: what `2` stands for?